server/v2/appmanager/appmanager.go

```diff
@@ -12,6 +12,45 @@ import (
 	"cosmossdk.io/core/transaction"
 )
 
+// AppManager is a coordinator for all things related to an application
+// It is responsible for interacting with stf and store.
+// Runtime/v2 is an extension of this interface.
+type AppManager[T transaction.Tx] interface {
```

Comment:
if this is moved to cometbft we drop the dependency from cometbft on appamanger. This follows the design in which appmanager does not depend on stf as well

Reply:
we cannot really move it, but I can copy it to cometbft, sure!